packages/exchange/src/exchange/langfuse_wrapper.py

```diff
@@ -45,16 +42,6 @@ def auth_check() -> bool:
     return auth_val
 
 
-CURRENT_DIR = Path(__file__).parent
-PACKAGE_ROOT = find_package_root(CURRENT_DIR)
-
-LANGFUSE_ENV_FILE = os.path.join(PACKAGE_ROOT, ".env.langfuse.local") if PACKAGE_ROOT else None
```

Comment:
i didn't mind this approach, if feel what we could've improved here is deferred the loading so it wasn't during module init. moving everything to a environment variable just pollutes how we manage configs more

personally i feel approach here is to configure them in `$XDG_CONFIG_HOME` to centralize it.